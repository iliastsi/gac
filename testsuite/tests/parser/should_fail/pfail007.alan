hello() : proc
  foo(f1:int  : proc
  {}
{
}
