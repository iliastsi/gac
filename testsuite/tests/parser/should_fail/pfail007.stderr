pfail007.alan:2:15: Error: Parse error on input `:'
	Probably unclosed `)'

